File: app/src/at-myhuynh/java/asiantech/internship/summer/java_core/ex18/CountWord.java
package asiantech.internship.summer.java_core.ex18;

import java.util.logging.Level;
import java.util.logging.Logger;

import asiantech.internship.summer.java_core.common.Common;

public class CountWord {

    public static void main(String[] args) {
        Logger log = Logger.getLogger(CountWord.class.getName());
        String strWord = Common.inputString("Nhập vào chuỗi: ");
        int count = strWord.split("[ ]+").length;
        if (log.isLoggable(Level.INFO)) {
            log.info("'" + strWord + "' có " + count + " từ.");
        }
    }

}

File: app/src/at-myhuynh/java/asiantech/internship/summer/kotlin/ex18/CountWord.kt
package asiantech.internship.summer.kotlin.ex18

import asiantech.internship.summer.kotlin.common.Common

object CountWord {

    @JvmStatic
    fun main(args: Array<String>) {
        val strWord = Common.inputString("Nhập vào chuỗi: ")
        val count = strWord.split("\\s+".toRegex()).dropLastWhile { it.isEmpty() }.size
        println("'$strWord' có $count từ.")
    }

}
